packages/cli/src/lib/deploy/Deployer.ts

```diff
@@ -34,14 +34,14 @@ export class Deployer {
     deployModulesDir: "modules/",
   };
   public manifest: DeployManifest;
-  private _config: DeployerConfig;
+  public readonly config: DeployerConfig;
```

Comment:
Instead of making the config public, requiring us to "lock down" the structure by adding readonly specifiers, I think we should instead just expose getter methods to the `Deployer` class for these internal config properties.

Reply:
yeah, that makes way more sense. thanks!!